teachertool/src/components/HeaderBar.tsx

```diff
@@ -3,67 +3,112 @@ import * as React from "react";
 import { Button } from "react-common/components/controls/Button";
 import { MenuBar } from "react-common/components/controls/MenuBar";
 
-interface HeaderBarProps {
-}
+interface HeaderBarProps {}
 
-export class HeaderBar extends React.Component<HeaderBarProps> {
-    protected reportAbuseUrl = "https://github.com/contact/report-content";
+export const HeaderBar: React.FC<HeaderBarProps> = () => {
+    const appTheme = pxt.appTarget?.appTheme;
 
-    protected getOrganizationLogo(targetTheme: pxt.AppTheme) {
-        const logoUrl = targetTheme.organizationWideLogo;
-        return <div className="header-logo">
-            {logoUrl
-                ? <img src={logoUrl} alt={lf("{0} Logo", targetTheme.organization)}/>
-                : <span className="name">{targetTheme.organization}</span>}
-        </div>
-    }
+    const brandIconClick = () => {};
```

Comment:
Are there any functional changes in this file, or mostly formatting? Just want to know where to focus :)

Reply:
Mostly adjusting css classes. I ran prettier on it and the reformatting was significant. I plan to do a prettier-only PR after this to fix formatting across the project. Then we can make a practice of running it as part of each PR.

Reply:
If we must use prettier, I will do my best. But to my personal taste, I think it actually makes the code harder to read...

Reply:
I don't love prettier, but I do feel it's important to have consistent styling. I wonder how configurable prettier is. I don't like how it stacks lines. Maybe we can increase its max line width to compensate that.

Reply:
Yeah, that would be a big help. I vote we investigate that before doing any big prettier-only PR :) Seems like it ought to be possible: https://prettier.io/docs/en/options.html#print-width